File: Demo/D7/Unit2.pas
unit Unit2;

interface

uses
  Windows, Messages, SysUtils, Variants, Classes, Graphics, Controls, Forms,
  Dialogs, StdCtrls, Buttons, ExtCtrls;

type
  TfmDBLogin = class(TForm)
    Label2: TLabel;
    Label1: TLabel;
    edPassword: TEdit;
    Panel1: TPanel;
    BitBtn1: TBitBtn;
    BitBtn2: TBitBtn;
    edUserName: TEdit;
    procedure FormClose(Sender: TObject; var Action: TCloseAction);
  private
    { Private declarations }
  public
    { Public declarations }
    cPassword, cUserName:String;
  end;

var
  fmDBLogin: TfmDBLogin;

implementation

{$R *.dfm}

procedure TfmDBLogin.FormClose(Sender: TObject; var Action: TCloseAction);
begin
  if ModalResult = mrOk then begin
    cUserName := Trim(edUserName.Text);
    cPassword := Trim(edPassword.Text);
  end;
end;

end.

File: Demo/BCB6/Unit2.cpp
//---------------------------------------------------------------------------

#include <vcl.h>
#pragma hdrstop

#include "Unit2.h"
//---------------------------------------------------------------------------
#pragma package(smart_init)
#pragma resource "*.dfm"
TfmDBLogin *fmDBLogin;
//---------------------------------------------------------------------------
__fastcall TfmDBLogin::TfmDBLogin(TComponent* Owner)
        : TForm(Owner)
{
}
//---------------------------------------------------------------------------
void __fastcall TfmDBLogin::FormClose(TObject *Sender, TCloseAction &Action)
{
  if (ModalResult == mrOk) {
    cUserName = edUserName->Text.Trim();
    cPassword = edPassword->Text.Trim();
  }
}
//---------------------------------------------------------------------------
